portfolio/src/main/webapp/index.html

```diff
@@ -1,39 +1,65 @@
 <!DOCTYPE html>
 <html>
-	<head>
-		<meta charset="UTF-8">
-		<title>My Portfolio</title>
-		<link rel="stylesheet" href="style.css">
-		<script src="script.js"></script>
-	</head>
-	<body>
-		<div id="content">
-			<h1>Jenny's Portfolio</h1>
-
-			<table>
-				<th onclick="Toggle('main-section')">home</th>
-				<th onclick="Toggle('about-section')">about me</th>
-				<th onclick="Toggle('contact-section')">contact me</th>
-			</table>
-
-				
-			<div id="main-section">
-				<img id="headshot" src="/images/Headshot_Jenny.JPG" alt="Image of me."/>
-				<p>Click here to see a random fact about me!:</p>
-				<button onclick="addRandomGreeting()">Hello</button>
-				<div id="greeting-container"></div>
+
+<head>
+	<meta charset="UTF-8">
+	<title>My Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body class="upper-section">
+	<div id="content">
+		<h1>Jenny Baik</h1>
+
+		<table id="menu">
+			<th onclick="Toggle('main-section')">home</th>
+			<th onclick="Toggle('about-section')">about me</th>
+			<th onclick="Toggle('contact-section')">contact me</th>
+		</table>
+
+
+		<div id="main-section">
+			<img id="headshot" src="/images/Headshot_Jenny.JPG" alt="Image of me."/>
 			</div>
 
 			<div id="contact-section">
-				<button link="https://www.instagram.com/waikandbaik/">Visit my food instagram!</button>
-				<button link="https://www.linkedin.com/jennybaik206">Connect with me on LinkedIn!</button>
-				<button link="mailto:baikj@google.com">Email me!</button>
+				<img src="/images/glyph-logo_May2016.png" usemap="#instagram"/>
+				<map name="instagram">
+					<area shape="rect" coords="0,0,30,30" href="https://www.instagram.com/waikandbaik/"/>
+				</map>
```

Comment:
Why use a shape map here? (If you're making the whole image a link, you can alternatively wrap the <img> in an <a href>)